pkg/handler/collector/github/READMD.md

```diff
@@ -0,0 +1,51 @@
+# GitHub Collector README
+
+## Overview
+
+The GitHub Collector is a component of the GUAC project designed to download metadata documents stored in GitHub releases or workflow artifacts.
+
```

Comment:
@nathannaveen I would just add a not here that `guacIngest` (along with a pubsub and blob store) are needed for `guacCollect` to run. Does not have to be fully explained but just a warning.

Reply:
Were you thinking of putting it in the prerequisites, or a separate section?

Reply:
just add a **note**

Reply:
Added!